model/src/pyrenew/deterministic/process.py

```diff
@@ -2,6 +2,7 @@
 
 import jax.numpy as jnp
 from pyrenew.deterministic.deterministic import DeterministicVariable
+from pyrenew.metaclass import SampledValue
 
 
 class DeterministicProcess(DeterministicVariable):
```

Comment:
I'd also suggest reviewing the implementation here. I think given some other choices we've made we'll perhaps want to modify it. Out of scope for this PR, however.

Reply:
Agreed